src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -60,6 +60,8 @@
   private boolean m_needToRecordBattleStatistics = true;
   private boolean m_needToCheckDefendingPlanesCanLand = true;
   private boolean m_needToCleanup = true;
+  private boolean m_needToDoRockets = true;
```

Comment:
Wouldn't that make needToDoRockets inconsistent with all the other private boolean attributes in this class?